Tools/clinic/clinic.py

```diff
@@ -4422,18 +4501,27 @@ class ParamState(enum.IntEnum):
     RIGHT_SQUARE_AFTER = 6
 
 
+VersionTuple = tuple[int, int]
```

Comment:
I'd prefer it if this could be defined above the `Parameter` class, for greater readability, since it's used inside the `Parameter` class